docs/metrics/work-in-progress.md

```diff
@@ -1,5 +1,10 @@
 # Work in progress metrics
 
+Pulse calculates the work in progress metrics based on the status of all pull requests targeting any base branch in your GitHub repositories.
+
+!!! important
+    You must install the [GitHub integration](../one-click-integrations.md#github) for Pulse to calculate the work in progress metrics.
+
```

Comment:
It doesn't hurt mentioning that Pulse can only calculate these metrics if the GitHub integration is installed.